src/ObjectPool/src/DefaultPooledObjectPolicy.cs

```diff
@@ -18,8 +18,11 @@ public override T Create()
     /// <inheritdoc />
     public override bool Return(T obj)
     {
-        // DefaultObjectPool<T> doesn't call 'Return' for the default policy.
```

Comment:
BTW, this comment became untrue when I revamped the implementation of DefaultObjectPool a little while ago.

Reply:
#45251 for reference.